roles/delivery_manager.md

```diff
@@ -0,0 +1,63 @@
+
+### Nuture highly performing teams
+
+We expect our Delivery Managers to nurture a culture of high performance in their teams. Where customers do not provide adequate pressure to deliver, we expect our Delivery Managers to constructively challenge our teams. It is important that we’re doing the right thing for our customers: delivering valuable, working software as expediently as we’re able
+
+### Manage risks and budgets
+
+We expect our Delivery Managers to proactively manage and mitigate risks to our deliveries and customer relationships. We expect our Delivery Managers to ensure the customer is delighted with what we deliver for their budget.
+
+### Define, capture and share necessary reporting materials
+
+We expect our Delivery Managers to work with customers to the define the frequency and type of reporting that will be most valuable to them.
+
+### Ensure teams and individuals take responsibility for delivery quality
+
+We expect our Delivery Managers to ensure teams are delivering with acceptable levels quality. We expect Delivery Managers to encourage their team to identify and proactively 'pull' from others when they need testing support. We expect Delivery Managers to *not* provide a safety net to their team by automatically testing features.
+
+## Delivery Lead Expectations
+
+### Be the point of contact for the customer
+
+We expect our Delivery Leads to naturally position themselves as the day to day point of contact for the customer. A Delivery Lead is expected to gain the trust of the customer such that they’re seen as the first port of call for all customer contacts.
+
+### Ensure customer expectations are exceeded given the budget available
+
+We expect our Delivery Leads to understand the financial constraints, as well as the customer objectives for the delivery. A Delivery Lead is accountable for ensuring the customer’s expectations are exceeded for the available budget.
+
+### Ultimately accountable for the successful delivery of customer engagements
+
+We expect our Delivery Leads to assume ultimate responsibility for the successful delivery of customer engagements. The performance of the team and the delivery as a whole is understood as the Delivery Lead’s responsibility.
+
+### Provide accurate and honest reporting of Delivery Healthcheck to Directors
+
+We expect our Delivery Leads to proactively provide regular reporting to the Directors on the health of their delivery.
+
+### Ensure that at all times the team is delivering according to the Made Tech Delivery Standards
+
+We expect our Delivery Leads to be the guardians of the Made Tech Delivery Standards, ensuring the team understands and is proactively practising them. A Delivery Lead is expected to encourage conversation on both more widely evolving the Delivery Standards, and ahead of deciding to follow a different path on their delivery.
+
+### Ensure at all times that the customer would recommend Made Tech to their peers
```

Comment:
Wouldn't ensuring this at _all_ times possibly stop you from having honest conversations?

If you've got the option to sweep something under the carpet to keep the client happy in the short term, or have a bit of short term dent in your relationship to avoid a bigger problem later on then the directive shouldn't be recommendation at all costs.

Reply:
Oh, Jon. You and your good points.

Reply:
Nah, I'm just a pedant. I see where it's coming from after Xmas though - something about safeguarding reputation and client recommendations in the longterm maybe?

Reply:
No, I think you're very right. The reality is that sometimes you need to cause some upset in the short term for the better long term outcome